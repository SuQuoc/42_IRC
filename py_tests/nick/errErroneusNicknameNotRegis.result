:AfterLife 432   :Erroneus nickname
:AfterLife 432   :Erroneus nickname
:AfterLife 432   :Erroneus nickname
:AfterLife 001 clie :Welcome to the Internet Relay Network, user0
