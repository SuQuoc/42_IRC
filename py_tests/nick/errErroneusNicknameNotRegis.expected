:AfterLife 432 * clie,nt0 :Erroneous nickname
:AfterLife 432 * clie@nt0 :Erroneous nickname
:AfterLife 432 * clie!nt0 :Erroneous nickname
:AfterLife 001 client0 :Welcome to the Internet Relay Network, user0
:AfterLife 005 client0 NICKLEN=9 CHANTYPES=#& CHANMODES=oiklt :are supported by this server
